### DISCUSSION TITLE
Add Vertical Tabs

### DISCUSSION THREAD

**Discussion by philippilievski** [primefaces/community#3023] [PrimeVue]
I wanted to suggest adding vertical tabs to the Tabs component. Right now, we can only use horizontal tabs, but having vertical ones would be super helpful for layouts that need side navigation or have limited horizontal space.

I noticed you use vertical tabs in your own documentation, and they look and work great. It'd be awesome to have that feature available in the library.

Thanks for the consideration.

**Comment by j0rgedev** [comment]
Hey @philippilievski. I haven't seen vertical tabs in the docs, where did you see it? 🤔

**Reply by philippilievski** [reply]
> Hey @philippilievski. I haven't seen vertical tabs in the docs, where did you see it? 🤔

Hey @j0rgedev! Thanks for the quick reply.

This is a bit meta, but I noticed something interesting while inspecting the HTML of the docs page—specifically, a component referred to as a "tab."
![image](https://github.com/user-attachments/assets/a76d41b8-3f64-427c-980b-5d7d8d13f78b)


I'm curious about how this is implemented. I looked into existing options and found PrimeVue's Tabs component, but it only supports horizontal layouts: https://primevue.org/tabs/

I hope this makes sense 😄

**Reply by j0rgedev** [reply]
Ohhh I see. If you inspect that nav, it isn't using any primevue component since it's a built in component for the docs. As far as I know, there isn't support for vertical tabs. I'd love to do a PR for this but I guess this is a feature that the team has to do it 😁
